Servers/package.json

```diff
@@ -4,19 +4,21 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon index.js"
+    "start": "npm run build && node dist/index.js",
```

Comment:
Thank you for your attention to the details. 🙏🏻
For this part `"npm run build && node dist/index.js"` better to switch back to what you had already, I mean `"nodemon index.js"`.
But, let's not remove this one and maybe create another script like `"dist"` for `"npm run build && node dist/index.js"`